File: src/main/java/com/PrzeBarCore/Laboratorymanagementsystem/configuration/AwsSqsConfiguration.java
package com.PrzeBarCore.Laboratorymanagementsystem.configuration;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import software.amazon.awssdk.services.sqs.SqsClient;

@Configuration
public class AwsSqsConfiguration {
    @Bean
    public SqsClient sqsClient(){
        return SqsClient.create();
    }
}

File: order-monitoring-worker/src/main/kotlin/com/PrzeBarCore/ordermonitoringworker/configuration/AwsSqsConfiguration.kt
package com.PrzeBarCore.ordermonitoringworker.configuration

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Profile
import software.amazon.awssdk.services.sqs.SqsClient

@Profile("aws")
@Configuration
class AwsSqsConfiguration {
    @Bean
    fun sqsClient(): SqsClient = SqsClient.create()
}